pkgs/os-specific/linux/alsa-lib/default.nix

```diff
@@ -2,23 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "alsa-lib";
-  version = "1.2.4";
+  version = "1.2.5";
 
   src = fetchurl {
     url = "mirror://alsa/lib/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-91VL4aVs3/RotY/BwpuVtkhkxZADjdMJx6l4xxFpCPc=";
+    sha256 = "067ga0l6zr782kw8jdsqvbb20pcgnl0vkpnnz2n36fq8ii58k4lh";
   };
 
   patches = [
     ./alsa-plugin-conf-multilib.patch
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Ok, removed